sklearn/cluster/tests/test_affinity_propagation.py

```diff
@@ -185,7 +185,7 @@ def test_affinity_propagation_predict_non_convergence():
 def test_affinity_propagation_non_convergence_regressiontest():
     X = np.array([[1, 0, 0, 0, 0, 0], [0, 1, 1, 1, 0, 0], [0, 0, 1, 0, 0, 1]])
     af = AffinityPropagation(affinity="euclidean", max_iter=2, random_state=34).fit(X)
```

Comment:
We can catch the new coverage warning here using `pytest.warns` and check the warning message:

```python
af = AffinityPropagation(affinity="euclidean", max_iter=2, random_state=34)
with pytest.warns(ConvergenceWarning, match=...):
    af.fit(X)
```

Reply:
Done!